templates/midonet.repo.j2

```diff
@@ -0,0 +1,20 @@
+[{{ midonet_repo_repository_section_prefix }}]
+name={{ midonet_repo_repository_description_prefix }}
+baseurl={{ midonet_repo_repository_base_url }}/{{ midonet_repo_midonet_dir }}/{{ midonet_repo_midonet_release|default('stable') }}/el{{ ansible_distribution_major_version }}/
+enabled=1
+gpgcheck=1
+gpgkey={{ midonet_repo_repository_gpgkey_url }}
+
+[{{ midonet_repo_repository_section_prefix }}-openstack-integration]
```

Comment:
there is no MEM or non-nem openstack repo, but only one thing, name it [midonet-openstack-integration]
same goes for descriptions